api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto

```diff
@@ -805,16 +805,16 @@ message HttpFilter {
 
   // The name of the filter configuration. The name is used as a fallback to
   // select an extension if the type of the configuration proto is not
-  // sufficient. It also serves as a resource name in FilterConfigDS.
+  // sufficient. It also serves as a resource name in ExtensionConfigDS.
   string name = 1 [(validate.rules).string = {min_bytes: 1}];
 
   // Filter specific configuration which depends on the filter being instantiated. See the supported
   // filters for further documentation.
   oneof config_type {
     google.protobuf.Any typed_config = 4;
 
-    // [#not-implemented-hide:] Configuration source specifier for FilterConfigDS.
-    HttpFilterConfigSource filter_config_ds = 5;
+    // Configuration source specifier for ExtensionConfigDS.
+    HttpFilterConfigSource config_discovery = 5;
```

Comment:
Should we just have a `ExtensionConfigSource`?

Reply:
Done.